### PR TITLE
Add airports to matteocellucci-rome

### PR DESCRIPTION
**Pull request by matteocellucci**
With km distance from Rome.

Does anyone have any other suggestions?

### CODE REVIEW

File: matteocellucci/matteocellucci-rome.md [hunk 1]
```diff
@@ -7,3 +7,7 @@
 #### History
 - Visit Protestant cemetery
 - Visit Acqua Acetosa fountain
+
```
Comment (pbeaudry-processia) [inline matteocellucci/matteocellucci-rome.md:10]: Coliseum is not a good place to visit ?

Comment (scathaig) [inline matteocellucci/matteocellucci-rome.md:10]: sample comment - do not use :)

Comment (matteocellucci) [inline matteocellucci/matteocellucci-rome.md:10]: @pbeaudry-processia No, too much wind on the hair... You know, no windows! 💨

Comment (Framme) [inline matteocellucci/matteocellucci-rome.md:10]: @pbeaudry-processia what type of coliseum are you talking about? is it fun?

Comment (matteocellucci) [inline matteocellucci/matteocellucci-rome.md:10]: @Framme This one: [coliseum](https://images.duckduckgo.com/iu/?u=http%3A%2F%2Fwww.windoweb.it%2Fdesktop_italia%2Ffoto_lazio%2Ffoto_roma%2Ffoto_roma_029_colosseo.jpg&f=1)

**Pull request merged**